apis/cosmos-source.js

```diff
@@ -523,7 +523,7 @@ export default class CosmosAPI {
       chunk(trace.path.split('/'), 2).map(async ([port, channel]) => {
         const result = await this.get(
           `/ibc/core/channel/v1/channels/${channel}/ports/${port}/client_state`
-        )
+        ).catch((_) => undefined)
         return result.identified_client_state.client_state.chain_id
```

Comment:
So result is possibly undefined that cause error here?

Reply:
yes.... added checking below